getting_started.md

```diff
@@ -63,14 +67,36 @@ Since we simulated the ancestry of 20 *diploid* individuals, our tree sequence
 contains 40 *sample nodes*, one for each genome.
 :::
 
+(sec_processing_trees)=
+
 ## Processing trees
 
-A common idiom is to iterate over all the {class}`tree objects<tskit.Tree>` in a tree
+A common idiom is to iterate over all the {class}`tree objects<Tree>` in a tree
 sequence. This process underlies many tree sequence algorithms, including those we'll
 encounter later in this tutorial for calculating 
 {ref}`population genetic statistics<tskit:sec_stats>`.
 To iterate over your own tree sequence you can use
-{meth}`TreeSequence.trees()<tskit.TreeSequence.trees>`.
+{meth}`TreeSequence.trees`.
+
+::::{margin}
```

Comment:
As nice as this is on desktop, it breaks the mobile view pretty badly:
![Screenshot from 2021-06-21 23-43-23](https://user-images.githubusercontent.com/8552/122836675-95108480-d2ea-11eb-9d88-ac7cdb6e0b18.png)

Reply:
I've pushed a commit to move the warning downwards, which makes it a little better (although it obscures the next code box a tiny bit). I think this is, however, a JupyterBook graphical layout bug, as you can horizontally scroll the code boxes just fine. I'll open an issue on JupyterBook and hopefully it will eventually get fixed. Meanwhile I think this can be merged anyway, as it's still usable, especially with the new commit.

Reply:
Update - I added to an issue at https://github.com/executablebooks/jupyter-book/issues/1305 so hopefully it will be eventually fixed.